main/src/main/scala/sbt/Defaults.scala

```diff
@@ -3498,17 +3498,17 @@ object Classpaths {
     }
   )
 
-  def warnResolversConflict(ress: Seq[Resolver], log: Logger): Unit = {
-    val resset = ress.toSet
+  def warnResolversConflict(resolvers: Seq[Resolver], log: Logger): Unit = {
```

Comment:
Given that resolves is a key name in sbt, I don't think this change is good.

Reply:
Oh, is this `resolutions`?

Reply:
I meant `resolvers` not resolves, but what I mean is that within Defaults.scala, `Keys.*` are used without `Keys.`, so creating a parameter named `resolvers` would shadow `Keys.resolvers`. So in this particular case, using abbreviated `ress` I think was intentional, not a bug.

Reply:
If something isn't obvious, it's a poor choice. Would `resolverList` work?

Reply:
Sure. resolverList is fine.